app.py

```diff
@@ -21,6 +21,11 @@ def index():
 def get_remember_thee():
 	return jsonify(success=True)
 
+@app.route("/pray", methods=['POST'])
+def prayer_button():
```

Comment:
probably doesn't matter in terms of runnable code, but let's call this function `pray` rather than `prayer_button`